pages/volunteers.vue

```diff
@@ -25,5 +40,7 @@ const { client } = usePrismic()
 const { data: volunteers } = await useAsyncData('volunteers', () =>
   client.getSingle('volunteerspage')
 )
-usePrismicSEO(volunteers.value.data)
+if (volunteers.value?.data != null) {
```

Comment:
`usePrismicSEO(volunteers.value?.data)` Should be sufficient instead of if..else